x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/InvalidateApiKeyRequest.java

```diff
@@ -26,19 +28,24 @@
 
     private final String realmName;
     private final String userName;
-    private final String id;
     private final String name;
     private final boolean ownedByAuthenticatedUser;
+    private final String[] ids;
```

Comment:
Nit: Can we move this field up to where `id` used to be, so that the order matches serialization?